en/docs/guides/mfa/mfa-for-management-console.md

```diff
@@ -0,0 +1,97 @@
+# Configuring multi-factor authentication for the management console
```

Comment:
Can't we move this to the Management Console section under Setup?

Reply:
+1 will move it